### PR TITLE
[CTOR-654] refresh Azure Elastic Pool doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 1]
```diff
@@ -5,243 +5,608 @@ title: Azure Elastic Pool
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Cloud-Azure-Database-Elasticpool-custom" label="Cloud-Azure-Database-Elasticpool-custom">
+
+| Alias     | Modèle de service                                    | Description                                                           |
+|:----------|:-----------------------------------------------------|:----------------------------------------------------------------------|
+| Log-Usage | Cloud-Azure-Database-Elasticpool-Log-Api-custom      | Contrôle l'utilisation des fichiers et bases logs Elastic Pools Azure |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:33]: bases logs?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 2]
```diff
@@ -5,243 +5,608 @@ title: Azure Elastic Pool
...
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Azure-Database-Elasticpool-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
 
-### Objets supervisés
+| Alias | Modèle de service                               | Description                                              |
+|:------|:------------------------------------------------|:---------------------------------------------------------|
+| Cpu   | Cloud-Azure-Database-Elasticpool-Cpu-Api-custom | Contrôle l'utilisation CPU des Elastic Pools vCore Azure |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:44]: vCore?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 3]
```diff
@@ -5,243 +5,608 @@ title: Azure Elastic Pool
...
+| AZURERESOURCEGROUP | Set resource group (required if resource's name is used)                                                                   |                   | X           |
+| AZURESUBSCRIPTION  | Set Azure subscription (required if logged to several subscriptions)                                                       |                   | X           |
+| AZURETENANT        | Set Azure tenant ID                                                                                                        |                   | X           |
+| PROXYURL           | Proxy URL if any                                                                                                           |                   |             |
+| EXTRAOPTIONS       | Any extra option you may want to add to every command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                       |                   |             |
+
+> Paramétrez les options suivantes dans la macro EXTRAOPTIONS si vous superviser des ressources Microsoft Azure géré par 21Vianet (Azure China):
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:232]:
```suggestion
> Paramétrez les options suivantes dans la macro EXTRAOPTIONS si vous supervisez des ressources Microsoft Azure gérées par 21Vianet (Azure China):
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 4]
```diff
@@ -5,243 +5,608 @@ title: Azure Elastic Pool
...
 
-La commande devrait retourner un message de sortie similaire à: 
+La commande devrait retourner un message de sortie similaire à :
 
 ```bash
-OK: Elastic Pool 'SQLSRV001/SQLELP001' Statistic 'average' Metrics Workers usage percentage: 0.03 %, Sessions usage percentage: 0.00 % |
-'SQLSRV001/SQLELP001~average#elasticpool.workers.usage.percentage'=0.03%;;;; 'SQLSRV001/SQLELP001~average#elasticpool.sessions.usage.percentage'=0.00%;;;;
-Checking Pool'SQLSRV001/SQLELP001' 
-    Statistic 'average' Metrics Workers usage percentage: 0.03 %, Sessions usage percentage: 0.00 %
+OK: Allocated data storage percentage: 29 % Allocated data storage: 99 B Storage usage percentage: 2 % Storage usage: 21 B | 'elasticpool.storage.data.allocated.percentage'=29%;;;;'elasticpool.storage.data.allocated.bytes'=99B;;;;'elasticpool.storage.usage.percentage'=2%;;;;'elasticpool.storage.used.bytes'=21B;;;;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:389]: l'exemple est bon?

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 5]
```diff
@@ -5,238 +5,606 @@ title: Azure Elastic Pool
...
 
-Expected command output is shown below:
+The expected command output is shown below:
 
 ```bash
-OK: Elastic Pool 'SQLSRV001/SQLELP001' Statistic 'average' Metrics Workers usage percentage: 0.03 %, Sessions usage percentage: 0.00 % |
-'SQLSRV001/SQLELP001~average#elasticpool.workers.usage.percentage'=0.03%;;;; 'SQLSRV001/SQLELP001~average#elasticpool.sessions.usage.percentage'=0.00%;;;;
-Checking Pool'SQLSRV001/SQLELP001' 
-    Statistic 'average' Metrics Workers usage percentage: 0.03 %, Sessions usage percentage: 0.00 %
+OK: Allocated data storage percentage: 29 % Allocated data storage: 99 B Storage usage percentage: 2 % Storage usage: 21 B | 'elasticpool.storage.data.allocated.percentage'=29%;;;;'elasticpool.storage.data.allocated.bytes'=99B;;;;'elasticpool.storage.usage.percentage'=2%;;;;'elasticpool.storage.used.bytes'=21B;;;;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:387]: l'exemple est bon?

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 6]
```diff
@@ -5,243 +5,608 @@ title: Azure Elastic Pool
...
+| Macro              | Description                                                                                                                | Valeur par défaut | Obligatoire |
+|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AZURECLIENTID      | Set Azure client ID                                                                                                        |                   | X           |
+| AZURECLIENTSECRET  | Set Azure client secret                                                                                                    |                   | X           |
+| AZURECUSTOMMODE    | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | api               |             |
+| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:225]:
```suggestion
| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |      X      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 7]
```diff
@@ -5,243 +5,608 @@ title: Azure Elastic Pool
...
-| X         | AZURECUSTOMMODE   | Custom mode 'azcli'                                                                             |
-| X         | AZURESUBSCRIPTION | Subscription ID                                                                                 |
-| X         | AZURERESOURCE     | Id of the Elastic Pool resource (syntax: <server\_name\>/elasticpool/<elasticpools\_name\>)     |
+| Macro              | Description                                                                                                                | Valeur par défaut | Obligatoire |
+|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AZURECUSTOMMODE    | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | api               |             |
+| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:241]:
```suggestion
| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |      X      |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 8]
```diff
@@ -5,238 +5,606 @@ title: Azure Elastic Pool
...
-| X         | AZURECUSTOMMODE   | Custom mode 'azcli'                                                                             |
-| X         | AZURESUBSCRIPTION | Subscription ID                                                                                 |
-| X         | AZURERESOURCE     | Id of the Elastic Pool resource (syntax: <server\_name\>/elasticpool/<elasticpools\_name\>)     |
+| Macro              | Description                                                                                                                | Default value     | Mandatory   |
+|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AZURECUSTOMMODE    | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | api               |             |
+| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:241]:
```suggestion
| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |      X      |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md [hunk 9]
```diff
@@ -5,238 +5,606 @@ title: Azure Elastic Pool
...
+| Macro              | Description                                                                                                                | Default value     | Mandatory   |
+|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AZURECLIENTID      | Set Azure client ID                                                                                                        |                   | X           |
+| AZURECLIENTSECRET  | Set Azure client secret                                                                                                    |                   | X           |
+| AZURECUSTOMMODE    | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | api               |             |
+| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-elasticpool.md:225]:
```suggestion
| AZURERESOURCE      | Set resource name or ID (required)                                                                                         |                   |      X      |
```

**Review by lucie-dubrunfaut — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**